files-community/Files
# Fix: Fixed issue where column view jumps from left to right

**Resolved / Related Issues**
   Closes #11977

**Validation**
How did you test these changes?
   1. Open a folder using columns layout
   2. Navigate using arrow keys
